cms_form/templates/assets.xml

```diff
@@ -13,6 +13,8 @@ License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 		</xpath>
 
 		<xpath expr="//script[last()]" position="after">
+            <script type="text/javascript" src="/web/static/lib/py.js/lib/py.js"></script>
```

Comment:
note to myself: at some point we should add edit-specific assets to be loaded only in edit mode